projects/questionset-editor-library/src/lib/services/editor/editor.service.spec.data.ts

```diff
@@ -31,15 +31,15 @@ export const rootNode = {
                     contentDisposition: 'inline',
                     lastUpdatedOn: '2022-04-05T06:40:11.752+0000',
                     contentEncoding: 'gzip',
-                    showSolutions: false,
+                    showSolutions: 'No',
```

Comment:
@ramkumar-pacewisdom This is not as per new format please check the below document for the same - https://project-sunbird.atlassian.net/wiki/spaces/QB/pages/3299803150/Proposed+QuML+Spec+Changes?focusedCommentId=3316875648

Reply:
updated